api/v1alpha3/types.go

```diff
@@ -197,45 +173,10 @@ type IngressRule struct {
 	Destination *string `json:"destination,omitempty"`
 }
 
-// TODO
-// String returns a string representation of the ingress rule.
-/*
-func (i *IngressRule) String() string {
-	return fmt.Sprintf("protocol=%s/range=[%d-%d]/description=%s", i.Protocol, i.FromPort, i.ToPort, i.Description)
-}
-*/
-
 // IngressRules is a slice of Azure ingress rules for security groups.
 type IngressRules []*IngressRule
 
-// TODO
-// Difference returns the difference between this slice and the other slice.
-/*
-func (i IngressRules) Difference(o IngressRules) (out IngressRules) {
-	for _, x := range i {
-		found := false
-		for _, y := range o {
-			sort.Strings(x.CidrBlocks)
-			sort.Strings(y.CidrBlocks)
-			sort.Strings(x.SourceSecurityGroupIDs)
-			sort.Strings(y.SourceSecurityGroupIDs)
-			if reflect.DeepEqual(x, y) {
-				found = true
-				break
-			}
-		}
-
-		if !found {
-			out = append(out, x)
-		}
-	}
-
-	return
-}
-*/
-
 // PublicIP defines an Azure public IP address.
-// TODO: Remove once load balancer is implemented.
```

Comment:
**resolved**: load balancer is already implemented but we still reference this type in the `Network` struct for `APIServerIP`.